public/pages/utils/helpers.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { CatIndex, IndexAlias } from '../../../server/models/types';
+import sortBy from 'lodash/sortBy';
+import { DetectorListItem } from '../../models/interfaces';
+import { SORT_DIRECTION } from '../../../server/utils/constants';
+import { ALL_INDICES, ALL_DETECTOR_STATES, DETECTOR_STATES } from './constants';
+
+export function sanitizeSearchText(searchValue: string): string {
+  if (!searchValue || searchValue == '*') {
+    return '';
+  }
+  if (canAppendWildcard(searchValue)) {
+    return `*${searchValue}*`;
+  } else {
+    return searchValue;
+  }
+}
+function canAppendWildcard(searchValue: string): boolean {
+  // If it's not a letter or number, reject it
+  if (!searchValue || !/[a-z0-9]/i.test(searchValue)) {
+    return false;
+  }
+  return true;
+}
+
+const isSystemIndices = (index: string) => {
```

Comment:
minor: From this method name, will return `true` if index is system indices (start with '.'), but it will return `false` for system indices. How about we rename this method  or make it return true for system indices?

Reply:
I see, this was existing code I glossed over. Will change to isUserIndex().